build.gradle

```diff
@@ -90,3 +90,15 @@ tasks.withType(JavaCompile).configureEach {
 		 it.options.release = targetVersion
 	}
 }
+
+java {
+	def javaVersion = JavaVersion.toVersion(targetJavaVersion)
+	if (JavaVersion.current() < javaVersion) {
+		toolchain.languageVersion = JavaLanguageVersion.of(targetJavaVersion)
+	}
+	archivesBaseName = project.archives_base_name
+	// Loom will automatically attach sourcesJar to a RemapSourcesJar task and to the "build" task
+	// if it is present.
+	// If you remove this line, sources will not be generated.
+	withSourcesJar()
+}
```

Comment:
what's the reason for adding this section?

Reply:
looks copied from the mcdev template, though the `withSourcesJar()` isn't a bad idea

Reply:
This is a bad behavior because the fabric room will remap the source code, and I feel like my behavior is foolish